dependency's AAR metadata (META-INF/com/android/build/gradle/aar-metadata.properties) is greater than this module's compileSdkVersion (android-30). Dependency: androidx.appcompat:appcompat:1.4.1.

Please help I can't run app


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckAarMetadataWorkAction
   > The minCompileSdk (31) specified in a
     dependency's AAR metadata (META-INF/com/android/build/gradle/aar-metadata.properties)
     is greater than this module's compileSdkVersion (android-30).
     Dependency: androidx.appcompat:appcompat:1.4.1.
     AAR metadata file: /Users/padepokan79/.gradle/caches/transforms-3/f04c5eb52e84d3d415c26e0192e2f0e8/transformed/appcompat-1.4.1/META-INF/com/android/build/gradle/aar-metadata.properties.



I found this problem on the react-native  issues (https://github.com/facebook/react-native/issues/35249 and https://github.com/facebook/react-native/issues/35210),   I change react-native version 0.66.1 to 0.66.5 in package.json
and  I run npm install I got I got an error
![image](https://user-images.githubusercontent.com/17938801/200534926-7742f897-5151-4e01-968b-93ba875527c9.png)
How can I fix this. thanks

--- a/app/package.json
+++ b/app/package.json
@@ -39,7 +39,7 @@
     "query-string": "7.0.1",
     "react": "17.0.2",
     "react-i18next": "11.13.0",
-    "react-native": "0.66.1",
+    "react-native": "0.66.5",
     "react-native-animated-pagination-dots": "0.1.72",
     "react-native-argon2": "2.0.1",
     "react-native-bouncy-checkbox": "^3.0.5",

--- a/core/package.json
+++ b/core/package.json
@@ -46,7 +46,7 @@
     "query-string": "^7.0.1",
     "react": "17.0.2",
     "react-i18next": "^11.13.0",
-    "react-native": "0.66.1",
+    "react-native": "0.66.5",
     "react-native-animated-pagination-dots": "^0.1.72",
     "react-native-argon2": "^2.0.1",
     "react-native-bouncy-checkbox": "^3.0.5",

